CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.2
+  - Fix concurrency issues in combination with Logstash 6.x and multiple worker threads
```

Comment:
I suggest linking https://github.com/jruby/jruby/issues/4868 for more information.

Reply:
@jsvd good idea :)